src/pages/overview/pdf-extract-api/gettingstarted.md

```diff
@@ -323,7 +323,7 @@ Jumpstart your development by bookmarking or downloading the following key resou
 
 The samples project requires the following:
 
--   NET: version 6.0 or above
+-   NET: version 8.0 or above
 -   A build Tool: Either Visual Studio or .NET Core CLI.
 
 #### Authentication
```

Comment:
line 333 -> .NET Samples instead of `adobe-dc-pdf-services-sdk-java-samples`

Reply:
Fixed